app/models/comment.rb

```diff
@@ -0,0 +1,4 @@
+class Comment < ApplicationRecord
+  belongs_to :post
+  validates :body, presence: true, length: { maximum: 100 }
+end
```

Comment:
- [ ] Comments should belong to both post and user 👇 
![Screen Shot 2020-11-03 at 11 41 10 PM](https://user-images.githubusercontent.com/48804258/98048483-0ec43280-1e2e-11eb-9d13-f3428e87fbf3.png)